docs/docs/howto/encrypted-storage.md

```diff
@@ -9,177 +9,152 @@ This step is recommended for any Contrast deployment that stores sensitive data
-  FMC:         None
-Platform TCB: TCB Version:
-  Microcode:   72
-  SNP:         23
-  TEE:         0
-  Boot Loader: 9
-  FMC:         None
+contrast generate resources/
+kubectl apply -f resources/
+contrast set resources/
 ```
 
-Use the values from `Platform TCB` to fill in the `MinimumTCB` values in the generated `manifest.json` file.
+## Verifying the deployment
 
-:::note[Attention!]
+In many scenarios, users may require assurance of an application's security and integrity before interacting with it—for example, prior to submitting sensitive data or casting a vote.
+Users of an app may want to verify its security and identity before sharing data by writing it to the persistent volume.
+Please see [this section](../getting-started/deployment.md#7-verify-deployment) of the documentation for more information on this topic.
 
-This must be done on a trusted machine, with a secure and trusted connection to it.
+## Example application: MySQL
 
-:::
-
-</TabItem>
-<TabItem value="metal-qemu-tdx" label="Bare metal (TDX)">
-
-```sh
-contrast generate --reference-values metal-qemu-tdx resources/
-```
-
-On bare-metal TDX, `contrast generate` is unable to fill in the `MrSeam` value as it depends on your platform configuration.
-It will have to be filled in manually.
-
-`MrSeam` is the SHA384 hash of the TDX module. You can retrieve it by executing
+The above guide can be tried out with a [MySQL](https://mysql.com) sample deployment which is part of the contrast release.
+You can add the deployment file to your resources by running:
 
 ```sh
-sha384sum /boot/efi/EFI/TDX/TDX-SEAM.so | cut -d' ' -f1
+curl -fLO https://github.com/edgelesssys/contrast/releases/latest/download/mysql-demo.yml --output-dir resources
 ```
 
-:::note[Attention!]
 
-This must be done on a trusted machine, with a secure and trusted connection to it.
+MySQL is an open-source database used to organize data into
+tables and quickly retrieve information about its content. All of the data in a
+MySQL database is stored in the `/var/lib/mysql` directory. In this example, we
+use the workload secret to setup an encrypted LUKS mount for the
+`/var/lib/mysql` directory to easily deploy an application with encrypted
+persistent storage using Contrast.
 
-:::
+The resources provided in this demo are designed for educational purposes and
+shouldn't be used in a production environment without proper evaluation. When
+working with persistent storage, regular backups are recommended in order to
+prevent data loss. For confidential applications, please also refer to the
+[security considerations](./hardening.md). Also be
+aware of the differences in security implications of the workload secrets for
+the data owner and the workload owner. For more details, see the [Workload
+Secrets](../architecture/secrets.md#workload-secrets) documentation.
 
-</TabItem>
-</Tabs>
+The MySQL deployment is defined as a `StatefulSet`, with a storage setup analogous to what has been described above.
```

Comment:
The MySQL client should be mentioned, too: it is referred to in the callout below.